libraries/browser-tracker-core/src/tracker/types.ts

```diff
@@ -580,6 +580,11 @@ export interface BrowserTracker {
    */
   clearUserData: (configuration?: ClearUserDataConfiguration) => void;
 
+  /**
+   * Returns the currently used queue name or if the `eventMethod` argument is provided, the queue name for the eventMethod.
+   */
+  getQueueName: (eventMethod?: 'post' | 'get' | undefined) => string;
```

Comment:
nit: This can be `eventMethod?: 'post' | 'get'` if you are using `?`, right?

Reply:
This slipped in the PR, should be `Exclude<EventMethod, 'beacon'>` as in the implementation.